plugins/k8saudit-gke/OWNERS

```diff
@@ -0,0 +1,2 @@
+approvers:
+  - sboschman
```

Comment:
As per our [governance](https://github.com/falcosecurity/evolution/blob/main/GOVERNANCE.md#maintainership) adding a maintainer requires a [majority vote](https://github.com/falcosecurity/evolution/blob/main/GOVERNANCE.md#majority-vote) 

Eligible voters are @falcosecurity/plugins-maintainers 

So

/hold until we get the majority or a week has passed

Ofc, +1 from me :+1: